src/libs/ReportUtils.js

```diff
@@ -728,11 +730,6 @@ function getIcons(report, personalDetails, defaultIcon = null, isPayer = false)
         result.source = defaultIcon || Expensicons.FallbackAvatar;
         return [result];
     }
-    if (isConciergeChatReport(report)) {
-        result.source = CONST.CONCIERGE_ICON_URL;
-        result.name = CONST.EMAIL.CONCIERGE;
-        return [result];
-    }
```

Comment:
Why do we remove this?

Reply:
It fallback to `getIconsForParticipants()` at the end of `getIcons()` which gets `accountID` from `participantsList`.
We can keep this code here and add `accountID` getting logic but it's redundant.

Reply:
Niice, as long as it works i'm happy! :D